src/main.ts

```diff
@@ -41,6 +43,9 @@ function stackSuppressions(stacks: Stack[], suppressions: NagPackSuppression[])
   });
 }
 
+const iamRolePrefix = app.node.tryGetContext('rolePrefix');
```

Comment:
move to use CfnParameter instead of CDK context, which only works on deploying from source code

Reply:
done.